cmake/OpenCensusHelpers.cmake

```diff
@@ -38,6 +38,18 @@ function(opencensus_test NAME SRC)
   endif()
 endfunction()
 
+# Helper function like bazel's cc_test. Usage:
+#
+# opencensus_benchmark(trace_some_test internal/some_test.cc dep1 dep2...)
+function(opencensus_benchmark NAME SRC)
+  # if(BUILD_TESTING)
```

Comment:
IMO benchmarks should only be built if BUILD_TESTING is enabled. Please uncomment.

Reply:
ok, fixed